st3/sublime_lib/resource_path.py

```diff
@@ -0,0 +1,83 @@
+import sublime
+import os.path
+from collections import OrderedDict
+
+from .pure_resource_path import PureResourcePath
```

Comment:
I would actually define this class in the same file and remove `pure_resource_path`. I don't believe separating the classes makes sense when they are basically the same thing.